### ISSUE TITLE
答案的语言支持

### ISSUE THREAD

**Issue by jinsen47** [Nightonke/LeetCoder2#2]
我发现目前答案的语言都是C++的形式, 有支持其他语言的打算吗?

**Comment by Nightonke** [comment]
Yes, that's my plan. But I'm little busy these days. I will do this work as soon so possible.

**Comment by jinsen47** [comment]
I'm very fond of this app, I could offer some help if u need.

**Comment by Nightonke** [comment]
Yep, that would be so good if you give pull request~
